scripts/host-check

```diff
@@ -46,6 +60,9 @@ from typing import (
 )
 
 
+# pylint: enable=wrong-import-position
```

Comment:
I'd probably just put this directly following the final `import` - i.e. no newlines before it.

Reply:
`isort` for some reason, requires 2 newlines between the import statements and this. No clue why

Reply:
Ok, fine to leave as is then :-)